packages/vuetify/src/components/VOverlay/locationStrategies.ts

```diff
@@ -241,6 +241,11 @@ function connectedLocationStrategy (data: LocationStrategyData, props: StrategyP
   })
 
   watch([data.target, data.contentEl], ([newTarget, newContentEl], [oldTarget, oldContentEl]) => {
+    if (newContentEl === oldContentEl && Array.isArray(newTarget)) {
```

Comment:
@KaelWD probably needs to look this over

Reply:
[`6cb4874` (#15682)](https://github.com/vuetifyjs/vuetify/pull/15682/commits/6cb4874618509cd31e5f571185e89cb2a07e0d14)